src/remote.ts

```diff
@@ -386,7 +388,27 @@ export class Remote {
       // Probably just doesn't exist!
       sshConfigRaw = ""
     }
-    const parsedConfig = SSHConfig.parse(sshConfigRaw)
+
+    // We are going to extract only the coder config
+    let coderConfigRaw = ""
+    const startOfCoderConfig = sshConfigRaw.indexOf(startBlockComment)
+    const endOfCoderConfig = sshConfigRaw.lastIndexOf(endBlockComment)
```

Comment:
This (lastIndexOf) seems a bit unsafe considering there could be malformed comments. We should ensure there's only one of each exists or ask the user to correct it.

Reply:
A good one too!